src/content/components/TreeView.js

```diff
@@ -54,14 +54,25 @@ function reactStringWithHighlightedSubstrings(string, substring, className) {
   return result;
 }
 
+type TreeViewRowFixedColumnsProps = {
+  node: Node,
+  nodeId: IndexIntoFuncStackTable,
+  columns: Column[],
+  index: number,
+  selected: boolean,
+  onClick: (IndexIntoFuncStackTable, MouseEvent) => any,
```

Comment:
That's what I did initially and then changed them all to "any" as we don't really care about it. Maybe "mixed" is better than any, though.